chia/full_node/full_node.py

```diff
@@ -77,6 +78,9 @@ class FullNode:
     timelord_lock: asyncio.Lock
     initialized: bool
     weight_proof_handler: Optional[WeightProofHandler]
+    pending_tx_request: Dict[bytes32, bytes32]
```

Comment:
I understand the key is transaction ID and the value is peer ID. is it possible to create a type alias or (better yet) an actual type that is just a bytes32 object, for transaction ID and peer ID? I think it would aid readability and type-checking.

Reply:
correct on key&val, seems like extra work for not much